glTF/EXT_mesh_features/FeatureIdAttribute/README.md

```diff
@@ -0,0 +1,27 @@
+# Feature ID Attributes
+
+This sample demonstrates usage of the [`EXT_mesh_features`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features) extension for storing feature IDs for vertices of a mesh.
+
+The sample contains a glTF asset with a single mesh primitive. The mesh primitive consists of 4 quads (each formed by 2 triangles). The vertices in this mesh primitive have the usual `POSITION` and `NORMAL` vertex attributes. 
+
+### Feature IDs
+
+Additionally, each vertex has a _feature ID_. In this example, the feature ID is given as a [vertex attribute](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features#vertex-attribute). 
```

Comment:
I think you mean to this section? https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features#feature-id-by-vertex

Reply:
I actually had _removed_ all these `#anchor` links (but...  by searching for `.md#`, thus, overlooking this one - it should have been `...README.md#vertex-attributes` FWIW). The reason is that soon these Markdown files will no longer exist (and be replaced by `.adoc` files, with probably different anchor names). I'd like to reduce the churn and link rot that may happen due to the upcoming changes.